python/pyspark/ml/classification.py

```diff
@@ -857,6 +871,12 @@ def featureImportances(self):
         """
         return self._call_java("featureImportances")
 
+    @property
+    @since("2.0.0")
+    def trees(self):
+        """Trees in this ensemble. Warning: These have null parent Estimators."""
```

Comment:
Sure you aren't seeing the `RandomForestClassificationModel` method instead? The diff looks up to date on my end.